tests/scripts/thread-cert/Cert_5_2_04_REEDUpgrade.py

```diff
@@ -82,14 +97,80 @@ def test(self):
             time.sleep(5)
             self.assertEqual(self.nodes[i].get_state(), 'router')
 
-        self.nodes[REED].start()
+        # 2 DUT_REED: Attach to any node, 2-hops from the Leader.
+        self.nodes[DUT_REED].start()
         time.sleep(5)
-        self.assertEqual(self.nodes[REED].get_state(), 'child')
+        time.sleep(ROUTER_SELECTION_JITTER)
```

Comment:
Is it possible to sleep shorter? Maybe by setting router selection jitter in `setUp`?

Reply:
“Maybe by setting router selection jitter in setUp?”
？？
I wasn't setting jitter.  Here means wait for ROUTER_SELECTION_JITTER seconds and then I can test 3rd step.

Reply:
fixed